index.js

```diff
@@ -68,12 +68,27 @@ const normalizeLink = (allEntries, link, removeUnresolved) => {
   return resolvedLink;
 };
 
+const makeEntryObject = (item, itemEntryPoints) => {
+  const ownKeys = Object.keys(item);
```

Comment:
Why not an early return here? `if (!itemEntryPoints) { return item }` ❓

Reply:
Would avoid one `Array.filter` and one `Array.reduce`. Especially filter can be slow